js/stdlib/ts/shared_backend.ts

```diff
@@ -72,6 +72,17 @@ export const formatAssertInfo = (ai:any = null) => {
   return msg;
 };
 
+export const APIStateMismatchError = (bin: any, es: BigNumber, as: BigNumber) : Error => {
```

Comment:
Would you rather this be in `shared_impl.ts` so that I can type `bin` as `IBackend` instead of `any`?

Reply:
Oh, yea that's what I meant, sorry I'm dumb.